### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by sladyn98**
Closes : https://github.com/sladyn98/JCasC-VSCode-Plugin/issues/6

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/feature_request.md]: best to label it?
```suggestion
labels: 'feature'
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
labels: 'bug'
```

**Comment by sladyn98** [comment]
Yeah labeling it :)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (sladyn98) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Agreed :+1:

**Pull request merged**